Makefile

```diff
@@ -21,6 +21,6 @@ MLS.pdf: *.tex chapters/*.tex
 	pdflatex MLS.tex
 
 # Seems to be some issue with graphicpath, so set path here as well
-%.html: %.tex chapters/*.tex
+MLS.html: MLS.tex chapters/*.tex
```

Comment:
For things like this, I think it's good to leave a comment somewhere nearby explaining that % rules can't be used since they aren't supported by nmake.

Reply:
Ok, added that.